### PR TITLE
add: base readme.md for project

### PR DESCRIPTION
**Pull request by elit-altum**
Added a base README.md for the project with information about:
1. Basic overview of the project
2. Usage as a CLI and a newman reporter
3. Setting up the dev environment
4. Relevant badges and formatting

[Preview](https://jbt.github.io/markdown-editor/#nVVdjhNHEH6fU5R2lSgrMT9evIlYeR1tNgEtQigKoChaRXLPTHmmoadr6B8b88QNeEW85Bo5Ty6QHIHqHo8NZlGUWLJs91RXfX/dnrUTEEo2+uKoQu3QHM0T4NdjXHdCw4/CtiUJUyezvJ3Mk2TWTg/rZ3J+CQaFSp3skL/1ZPgBLMmAHto4goq0M6TuwEriGoSuocbSN2C8tkAaBPyK5bNrqMeJ2SyXc546DVP7W0HOZNeANdXFUetcb8/znBcy20pUtc0k5aWoG8xLL5X76rRYS9emD8VKPKmM7F3aGET9vXUbhRdLJVxqX3ph8GtFDV1oqjF7bv/fJKaeDtRTMkI3+KUpPVnHVf9tygqNlaTTIiuySVoqf0t77jjLe5bu+BguS/IuSQ4tBSE7G6zpDa1kjWzB3sUDM2DdyqqFivcrdLAhPxoarYyeuhah8sawQ2oD+Aor76RuQFhYo1LhM5T0BleSvN3VYA03g1i/fzPybtgqX2YVdflWIiVKmw9lJzE0GTxtJbcmr2poUfUR1RCqgElooTavMayaIWVLQx2TtAxKIaCue5LaxWJNTi43IIJSvMDQjOH4Dqw72bQOqGJyUHsTOAUi3DNLkgiCFXyOlYM1c4wZGOtuHhCFYU981/GRoCVcca7gtDid7Mna+JCWVYhcCGkTNw3sh842Pz47m353996kmEynxbS49+0J/PXuzd9/voWwA25+HmT6NwlP4J8/3r+JsXhmRYMjAVG94F/AX0vidmJ/jKOWYB1/CkUax9qgP96yDk68QMtRMRj4KuF11Y6a7QJ1J2jLwkQhx1EcKR3IhlCGB9vbYxD6MoP7bInUPFJx0PkEnCfJYrFIdN+Ny5A2213pblSsSX7I4FGE8imOcPUsFFVCtSzS+VlRTBbbrre2+S0knzEJZflOa4ksBrS1wI60fI0H3b0dmS/SdFe0YKYdN69jfj6u54jy/caKD7HWtIYS+XhZWbJYAWuwl939FHLOyK4yzlbUj+3aKzdquefxEbtYUJFSvIv15PuOR6QGDkgPB02yvFuHxvx/1jymJZzDAT9jFwE4l8UrQrK37JPz8coPO0fCWYzkT3oleaHjKwQsOt9H26+33oZ3jT1Xo64k2s/c3zn9ix+6Xz26jn9DXwjhtkE4kVleSp1Lbv0KZsI0dj40u4rN9LjVoWVkXjomxqFvhsOn+Brhij2eUBb3fwA=)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+B. Run the CLI for launching the dashboard
+
+```
+node ./bin/index <args>
+```
+
+C. Running the test suite along with linting
+
+```
+npm test
```
Comment (coditva) [inline README.md:65]: Maybe we can add all the different ways to run tests.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+npm install -g newman-dashboard
+```
+
+B. Launch the dashboard on `localhost:5001`
+
+```
+newman-dashboard
+```
+
+You can also choose to daemonize the dashboard using the `--daemonize` command.
```
Comment (coditva) [inline README.md]:
```suggestion
You can also choose to daemonize the dashboard using the `--daemonize` flag.
```

**Review by coditva — APPROVED** [review]
Looks good. Minor comments.

**Comment by elit-altum** [comment]
Made the changes! 🚀

**Pull request merged**